### ISSUE TITLE
Create scratch instance of Iris

### ISSUE THREAD

**Issue by steveellis** [culibraries/folio#170]
Now that we have repeatable infrastructure we need to create an environment which we can use to do more extensive testing of things without bringing down the "production" version that people are using.

I think the steps are:

- [x] Determine if we need a branch for this work. We may not since it should all be handled in pulumi.
- [x] Work out what to do with the secrets. It is likely that there are going to be separate secrets for each stack.
- [x] Figure out what to do with the database. We're going to want a the test database to be separate from the production instance but copy the current production data over. This will likely require some changes to index.ts where we create the db cluster from scratch.
- [x] Create a new pulumi stack called something like `test` or `scratch`.
- [x] Run `pulumi up`. Test.
- [x] Map new domains. Maybe something like `folio-iris-scratch.cublcta` etc.

**Comment by steveellis** [comment]
I think we're gravitating toward calling this env `scratch` since the word "test" is used fairly broadly. This will also clearly distinguish it from a staging environment, which should probably be something distinct which can be swappable with production.

**Comment by steveellis** [comment]
I think this is what we DON'T want to do: https://github.com/pulumi/pulumi/pull/5062. Because we don't to copy the config at all. Instead to create the new stack we should just need (and want) `pulumi stack init`. Then create each secret individually with new values with `pulumi config set <secret name> --secret` for secrets and `pulumi config set <config name>`.

These are our current configs that would need to be set manually in the new stack:
```
aws:region          us-west-2
awsAccountId        [secret]
db-admin-password   [secret]
db-admin-user       [secret]
db-host             [secret]
db-user-name        [secret]
db-user-password    [secret]
superuser-name      [secret]
superuser-password  [secret]
```

Everything else is dynamically generated at runtime.

`pulumi stack -i` shows all the items that are exported from the stack. But those should mostly be left alone.

This stack that we have is called `dev `. Can we rename it `production`? Yes `pulumi stack rename`.

How to switch between stacks: `pulumi stack select`.

Should also update the readme with this, so that argues for creating a branch.

**Comment by steveellis** [comment]
@jwfuller - In working on this I noticed that the way I was doing our stacks via s3 wasn't quite right. I had in the readme instructions to login like this:
```
pulumi login s3://cubl-pulumi/folio/<stack name>
```

The stack name shouldn't have been included in the path. Rather it should have just been
```
pulumi login s3://cubl-pulumi/folio
```

I noticed `pulumi stack ls` was only including the current stack rather than all the stacks. I re-arranged s3 to fix this. Anyway just something to be aware of. Logging in should connect you to all the stacks, not just one. Now creating a stack works fine after logging in. What we had previously was fine if we had only one stack.
